src/js/helpers/graph/declarationHelper.mock.js

```diff
@@ -0,0 +1,374 @@
+/*
+ * Copyright (C) 2020 Knot.x Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export const singleNode = {
+  id: 'rootId', label: 'A', status: 'success', type: 'single',
+};
+
+export const compositeNode = {
+  id: 'rootId',
+  label: 'Test',
+  status: 'error',
+  type: 'composite',
+  subtasks: [
+    {
+      id: 'sub1', label: 's1', status: 'success', type: 'single',
+    },
+    {
+      id: 'sub2', label: 's2', status: 'success', type: 'single',
+    },
+  ],
+};
+
+export const compositeWithTransitions = {
+  id: 'rootId',
+  label: 'Test',
+  status: 'error',
+  type: 'composite',
+  subtasks: [
+    {
+      id: 'sub1',
+      label: 's1',
+      status: 'success',
+      type: 'single',
+    },
+    {
+      id: 'sub2',
+      label: 's2',
+      status: 'success',
+      type: 'single',
+      on: {
+        _error: {
+          id: 'sub2-1', label: 's2-1', status: 'success', type: 'single',
+        },
+      },
+      response: {
+        transition: '_error',
+      },
+    },
+  ],
+  on: {
+    _success: {
+      id: 'last',
+      label: 'Test2',
+      status: 'error',
+      type: 'single',
+    },
+  },
+  response: {
+    transition: '_success',
+  },
+};
+
+export const nodeWithMultipleTransitions = {
+  id: 'someId',
+  label: 'Root',
+  status: 'success',
+  type: 'single',
+  on: {
+    _success: {
+      id: 'trans1',
+      label: 'Trans 1',
+      status: 'unprocessed',
+      type: 'single',
+    },
+    _error: {
+      id: 'trans2',
+      label: 'Trans 2',
+      status: 'unprocessed',
+      type: 'single',
+    },
+    custom1: {
+      id: 'trans3',
+      label: 'Trans 3',
+      status: 'success',
+      type: 'single',
+    },
+    custom2: {
+      id: 'trans4',
+      label: 'Trans 4',
+      status: 'unprocessed',
+      type: 'single',
+    },
+  },
+  response: {
+    transition: 'custom1',
+  },
+};
+
+export const compositeWithUnprocessedSubtasks = {
```

Comment:
Which subtask is unprocessed? Maybe you thought about missing node scenario?

Reply:
I meant unprocessed child of a subtask. I'll change the name to make it clearer